apcd-cms/src/client/src/components/Admin/Extensions/AdminExtensions.tsx

```diff
@@ -1,19 +1,117 @@
-import React, { useEffect, useMemo } from 'react';
-import { useExtensions, ExtensionRow } from 'hooks/admin';
+import React, { useEffect, useState } from 'react';
+import { useExtensions, ExtensionRow, ExtensionEditRow } from 'hooks/admin';
+import LoadingSpinner from 'core-components/LoadingSpinner';
+import SectionMessage from 'core-components/SectionMessage';
+import Paginator from 'core-components/Paginator';
+import styles from './ExtensionList.module.css';
+import ViewExtensionModal from 'apcd-components/Extensions/ViewExtensionModal/ViewExtensionModal';
+import EditExtensionModal from 'apcd-components/Extensions/EditExtensionModal/EditExtensionModal';
 import { formatDate } from 'utils/dateUtil';
 
+
 export const AdminExtensions: React.FC = () => {
-  const { data, isLoading, isError } = useExtensions({});
+  const [status, setStatus] = useState('All');
+  const [org, setOrg] = useState('All');
+  const [page, setPage] = useState(1);
+  const { data, isLoading, isError, refetch } = useExtensions(status, org, page);
+  const [isViewModalOpen, setIsViewModalOpen] = useState(false);
+  const [isEditModalOpen, setIsEditModalOpen] = useState(false);
+
+    const [selectedExtension, setSelectedExtension] =
+    useState<ExtensionRow | null>(null);
+
+console.log(data);
+
+  const clearSelections = () => {
+    setStatus('');
+    setOrg('');
+    setPage(1);
+  };
 
-  if (isLoading) {
-    return <div>Loading...</div>;
+  useEffect(() => {
+    refetch();
+  }, [status, org, page, refetch]);
+
+   if (isLoading) {
+    <LoadingSpinner/>
   }
 
   if (isError) {
-    return <div>Error loading data</div>;
+          <SectionMessage type="error">
+            There was an error loading the page.{''}
+            <a href="https://txapcd.org/workbench/dashboard/tickets/create">
+              Please submit a ticket.
+            </a>
+          </SectionMessage>
```

Comment:
```suggestion
    return (
      <SectionMessage type="error">
        There was an error loading the page.{''}
        <a href="https://txapcd.org/workbench/dashboard/tickets/create">
          Please submit a ticket.
        </a>
      </SectionMessage>
      );
```

Reply:
Try testing by using locally` if (!isError) `or` if (!isLoading)` locally to test behavior. Without the return, they do not display.